crdt_commands.go

```diff
@@ -1,12 +1,17 @@
+// Copyright 2015 Basho Technologies, Inc. All rights reserved.
+// Use of this source code is governed by Apache License 2.0
+// license that can be found in the LICENSE file.
+
 package riak
 
 import (
 	"errors"
 	"fmt"
-	rpbRiakDT "github.com/basho-labs/riak-go-client/rpb/riak_dt"
-	proto "github.com/golang/protobuf/proto"
 	"reflect"
 	"time"
+
+	rpbRiakDT "github.com/basho-labs/riak-go-client/rpb/riak_dt"
+	proto "github.com/golang/protobuf/proto"
```

Comment:
This is strange, since running `go fmt` orders the imports differently. Is this part of your editor?